### PR TITLE
Dashify copyright_header.py

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
Works with updating and report, inserting works but normally results in there being two blocks of the 
```
// Distributed under the MIT software license, see the accompanying
// file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

### CODE REVIEW

File: contrib/devtools/copyright_header.py [hunk 1]
```diff
@@ -277,7 +279,7 @@ def exec_report(base_directory, verbose):
     $ ./copyright_header.py report <base_directory> [verbose]
 
 Arguments:
-    <base_directory> - The base directory of a bitcoin source code repository.
+    <base_directory> - The base directory of a dash source code repository.
```
Comment (UdjinM6) [inline contrib/devtools/copyright_header.py]:
```suggestion
    <base_directory> - The base directory of a Dash Core source code repository.
```

File: contrib/devtools/copyright_header.py [hunk 2]
```diff
@@ -408,32 +410,32 @@ def exec_update_header_year(base_directory):
...
+// Copyright (c) <year>-<lastModifiedYear> The Dash Core developers
 
 where the update is appropriate.
 
 Usage:
     $ ./copyright_header.py update <base_directory>
 
 Arguments:
-    <base_directory> - The base directory of a bitcoin source code repository.
+    <base_directory> - The base directory of a dash source code repository.
```
Comment (UdjinM6) [inline contrib/devtools/copyright_header.py]:
```suggestion
    <base_directory> - The base directory of Dash Core source code repository.
```

File: contrib/devtools/copyright_header.py [hunk 3]
```diff
@@ -548,14 +550,14 @@ def exec_insert_header(filename, style):
...
-If the file already has a copyright for "The Bitcoin Core developers", the
+If the file already has a copyright for "The Dash Core developers", the
 script will exit.
 
 Usage:
     $ ./copyright_header.py insert <file>
 
 Arguments:
-    <file> - A source file in the bitcoin repository.
+    <file> - A source file in the dash repository.
```
Comment (UdjinM6) [inline contrib/devtools/copyright_header.py]:
```suggestion
    <file> - A source file in the Dash Core repository.
```

**Comment by PastaPastaPasta** [comment]
Updated based on suggestions 👍

**Pull request merged**